### PR TITLE
Bug 1387640 - Remove exclusion profiles models/tables - part 3

### PR DESCRIPTION
**Pull request by camd**
Addresses the last of [Bug 1387640](https://bugzilla.mozilla.org/show_bug.cgi?id=1387640)

This removes the models and tables that were part of the Exclusion Profile feature which has been supplanted with the use of just Tiers.

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: treeherder/model/models.py [hunk 1]
```diff
@@ -21,7 +21,6 @@
 from django.forms import model_to_dict
 from django.utils import timezone
 from django.utils.encoding import python_2_unicode_compatible
-from jsonfield import JSONField
```
Comment (edmorley) [inline treeherder/model/models.py]: It looks like this is the last non-migrations usage of the `jsonfield` package, so after the next migrations squashing the package could be removed from the requirements file. Could you file a bug to remind us to remove it?

Also, the comment for the `NoOpSerializer` serializer says that it's only needed due to use of `JSONField`, perhaps we can remove it too?
https://github.com/mozilla/treeherder/blob/bde8e20f1f950c8edb953b1278b33575cc45da72/treeherder/webapp/api/serializers.py#L9-L14
(This is probably best in a separate bug/PR)

File: treeherder/model/models.py [hunk 2]
```diff
@@ -336,57 +335,6 @@ class Meta:
...
-    that you want to filter jobs on. These properties along with their values
-    are kept in the info field in json format
-    """
-    name = models.CharField(max_length=255, unique=True)
-    description = models.TextField(blank=True)
-    info = JSONField()
-    author = models.ForeignKey(User)
-
-    class Meta:
-        db_table = 'job_exclusion'
```
Comment (edmorley) [inline treeherder/model/models.py]: The `GRANT` entries for these tables can also be removed from `docs/admin.rst`

File: treeherder/model/models.py (hunk 1 above)
Comment (camd) [reply treeherder/model/models.py]: Sounds good.  Filed Bug 1399999 for this.

File: treeherder/model/models.py (hunk 2 above)
Comment (camd) [reply treeherder/model/models.py]: Dang!  I remembered seeing those in a couple of my searches, but somehow missed them.  Once again, nice catch...  :)

**Pull request merged**